client/www/scripts/modules/arc-user/arc-user.auth.factory.js

```diff
@@ -2,7 +2,7 @@
 
 var angBase = "/api";
 //var authBase = angBase;
-var authBase = "https://auth.strongloop.com/api";
+var authBase = "http://demo.strongloop.com:3001/api";
```

Comment:
good catch on the demo url. i don't know how that got in there.
i'll talk to raymond